### ISSUE TITLE
[Feature] Evenly space elements on a diagram

### ISSUE THREAD

**Issue by dermotobrien** [archimatetool/archi#189]
With multiple elements selected on a diagram, provide an option to space the elements evenly either vertically or horizontally. Leave the outer elements (topmost and bottommost for vertical or leftmost and rightmost for horizontal) in place and distribute the remainder with even spacing between the elements.

**Comment by dermotobrien** [comment]
I can have a go at implementing this if not already on the roadmap. Let me know.

**Comment by speytonhale** [comment]
I agree that this is a needed feature and would make a difference to diagrams. Most other products have such a feature in their alignment options.

**Comment by jbsarrodie** [comment]
Hi. In fact this is now easy to do through [jArchi](https://www.archimatetool.com/plugins/) (the script plugin for Archi). I personaly don't have time to do it now, but you should be able to do it quickly by yourself.

**Comment by robkamp** [comment]
I have created a gist to achieve this
